src/three/TilesRenderer.js

```diff
@@ -247,12 +262,55 @@ export class TilesRenderer extends TilesRendererBase {
 				} );
 
 			}
+			Promise.resolve().then( () => {
+
+				if ( this.ionAssetId ) {
+
+					const transform = this.root.cached.boxTransform;
+					let position = new Vector3().setFromMatrixPosition( transform );
+					let distance = position.length();
+
+					const surfaceDir = position.normalize();
+
+					const rotation = this.rotationBetweenDirections( surfaceDir, new Vector3( 0, 1, 0 ) );
+
+					// Rotate tiles to the north pole around the Earth
+					this.group.quaternion.x = rotation.x;
+					this.group.quaternion.y = rotation.y;
+					this.group.quaternion.z = rotation.z;
+					this.group.quaternion.w = rotation.w;
+
+					//Move tiles to the center of the Earth
+					this.group.position.y = -distance;
+
+				}
+
+			} );
 
 		} );
 		return pr;
 
 	}
 
+	/** set projection matrix to frustum for legacy versions of three.js */
+	setFromProjectionMatrix( frustum, m ) {
+
+		var planes = frustum.planes;
+		var me = m.elements;
+		var me0 = me[ 0 ], me1 = me[ 1 ], me2 = me[ 2 ], me3 = me[ 3 ];
+		var me4 = me[ 4 ], me5 = me[ 5 ], me6 = me[ 6 ], me7 = me[ 7 ];
+		var me8 = me[ 8 ], me9 = me[ 9 ], me10 = me[ 10 ], me11 = me[ 11 ];
+		var me12 = me[ 12 ], me13 = me[ 13 ], me14 = me[ 14 ], me15 = me[ 15 ];
+
+		planes[ 0 ].setComponents( me3 - me0, me7 - me4, me11 - me8, me15 - me12 ).normalize();
+		planes[ 1 ].setComponents( me3 + me0, me7 + me4, me11 + me8, me15 + me12 ).normalize();
+		planes[ 2 ].setComponents( me3 + me1, me7 + me5, me11 + me9, me15 + me13 ).normalize();
+		planes[ 3 ].setComponents( me3 - me1, me7 - me5, me11 - me9, me15 - me13 ).normalize();
+		planes[ 4 ].setComponents( me3 - me2, me7 - me6, me11 - me10, me15 - me14 ).normalize();
+		planes[ 5 ].setComponents( me3 + me2, me7 + me6, me11 + me10, me15 + me14 ).normalize();
+
+	}
```

Comment:
Which version of three.js are you using that doesn't include `setFromProjectionMatrix`?

Reply:
I had to use 0.111.0 for a project. I can remove this function if you wish.

Reply:
I see -- I'm a bit torn but I don't know if I want to start supporting every old version of three.js by copying code from the newer versions. Otherwise we'd have to do it for every function that gets added that we want to use. If you're using a version of three that doesn't support this function and you can't upgrade you should be able to patch it in like so:

```js
import { Frustum } from 'three';
Frustum.prototype.setFromProjectionMatrix = function () {
  /* ... */
};
```

Reply:
You are right. I'll remove it.